rollup.config.js

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* global __dirname */
+
+/**
+ * External dependencies
+ */
+import { resolve as resolvePath, dirname } from 'path';
+import resolve from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import { babel } from '@rollup/plugin-babel';
+import { terser } from 'rollup-plugin-terser';
+import svgr from '@svgr/rollup';
+import filesize from 'rollup-plugin-filesize';
+import css from 'rollup-plugin-import-css';
+import url from '@rollup/plugin-url';
+import json from 'rollup-plugin-json';
+import license from 'rollup-plugin-license';
+import workspacesRun from 'workspaces-run';
+
+/**
+ * Internal dependencies
+ */
+
+const plugins = [
+  resolve({
+    preferBuiltins: true,
+    dedupe: [],
+  }),
+  babel({
+    babelHelpers: 'inline',
```

Comment:
Trying to learn why `inline` is used as an option for [babelHelpers](https://github.com/rollup/plugins/tree/master/packages/babel#babelhelpers) here

Reply:
I used `runtime` initially as it was the recommendation, but I got some warnings when building.